docs/INSTALL.MD

```diff
@@ -1,18 +1,90 @@
-## Installation
+# Installation
 
 JuliaBox is developed and tested on Ubuntu 14.04. But it should be easy to adapt to any Linux environment supporting Docker.
 
-### Single Machine Development Setup
-1. Get JuliaBox sources:
-    - If `git` is not installed yet, run: `sudo apt-get install git`
-    - `git clone https://github.com/tanmaykm/JuliaDockerImages.git`
-    - `git clone https://github.com/JuliaLang/JuliaBox.git`
-2. Install docker and a few more packages required to develop and administer JuliaBox:
-    - Modify `JuliaBox/scripts/install/sys_install.sh` and set `DOCKER_FS` to `AUFS` instead of `DEVICEMAPPER`.
-        - This will avoid certain unnecessary configurations required only in certain production environments.
-    - Run `JuliaBox/scripts/install/sys_install.sh`
-3. Create work folders: `mkdir -p /jboxengine/conf /jboxengine/data /jboxengine/data/db`
-4. Configure JuliaBox:
+## Single-machine setup
+
+### Installation overview
+Installing JuliaBox on a single machine consists of five principle steps, the specifics of which vary depending upon the desired configuration. The following is a general overview of the process, and the key scripts and files involved. It may be useful to consult the included `Architecture.MD` document to gain a high-level overview of JuliaBox before reading these instructions.
+
+1. System preparation.
+
+	Preparation is performed using the `sys_install.sh` script included with the JuliaBox installation. After modification to specify the desired Docker configuration, the script performs the following tasks:
+	- installation of requisite system tools & libraries,
+	- installation of Docker, addition of current user to Docker group,
+	- installation of requisite Python libraries, and,
+	- configuration and launch of Docker.
+
+	Finally, the necessary work folders to store data & configuration items must be created.
+	
+2. JuliaBox configuration.
+
+	The `jbox.user` JuliaBox configuration file is created according to templates provided later in this document. This file sets:
+	- limits on the maximum number of active containers, and disks,
+	- details of administrators, and,
+	- the plugins to be used for this deployment.
+
+	The `jbox_configure.sh` script is executed to configure the JuliaBox session ID key, and move the server configuration templates to the work directories (from where they will be mounted by the Docker containers).
```

Comment:
The `jbox_configure.sh` script is executed to configure the JuliaBox installation secret (used as part of encryption key for signing and encrypting session data), and move...